src/BigQueryFunctions/CreateIssueTable.ts

```diff
@@ -0,0 +1,41 @@
+//Creates new table with Issue schema
+export default async function createIssueTable(  
+    datasetId: string, // Existing dataset
+    tableId: string, // Table to be created
+    schema = [
```

Comment:
Please have this schema definition in another file and pass that as argument. Please don't declare the default schema here in this function. that's not good.